releasenotes/notes/1.0/prepare-1.0-e13b9b8273055775.yaml

```diff
@@ -0,0 +1,70 @@
+---
+prelude: |
+  We are very pleased to release Qiskit 1.0.0. This release is the
+  culmination of 7 years of development to mature Qiskit into a stable high
+  performance SDK to perform quantum computation, and is the start of new
+  era for the Qiskit project and community. The most significant changes with the 1.0.0 
+ release relate to the stability policy, release cycles, and versioning.
+  Qiskit is now formally using `semantic versioning <https://semver.org/>`__ ,
```

Comment:
It seems that one of the lines is not indented like the others.

Reply:
I think it would be nice to have subsections for each of "stability policy, release cycles, and versioning" so readers can directly jump to what interest them most.

Reply:
```suggestion
  era for the Qiskit project and community. The most significant changes with the 1.0.0 
  release relate to the stability policy, release cycles, and versioning.
  Qiskit is now formally using `semantic versioning <https://semver.org/>`__ ,
```